cmd/auth/login.go

```diff
@@ -43,8 +43,36 @@ func newLoginCommand(persistentAuth *auth.PersistentAuth) *cobra.Command {
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		ctx := cmd.Context()
+
+		var profileName string
+		profileFlag := cmd.Flag("profile")
+		if profileFlag != nil && profileFlag.Value.String() != "" {
+			profileName = profileFlag.Value.String()
+		} else {
+			prompt := cmdio.Prompt(ctx)
+			prompt.Label = "Databricks Profile Name"
+			prompt.Default = persistentAuth.ProfileName()
```

Comment:
Previously it proposed default value based on host name, what does it prompts now?

Reply:
This should be just empty string, as we don't know where they are trying to authenticate to.